bin/setup

```diff
@@ -5,6 +5,9 @@ set -e
 gem install bundler --conservative
 bundle check || bundle install
 
+# Add binstubs to PATH via export PATH=".git/safe/../../bin:$PATH" in ~/.zshenv
+mkdir -p .git/safe
```

Comment:
I concur with Caleb.

Though if you are running a bin/setup file you haven't inspected, creating
.git/safe is the least of your worries.

On Friday, June 3, 2016, Caleb Thompson <notifications@github.com> wrote:

> In bin/setup
> <https://github.com/thoughtbot/scenic/pull/155#discussion_r65708461>:
>
> > @@ -5,6 +5,9 @@ set -e
> >  gem install bundler --conservative
> >  bundle check || bundle install
> >
> > +# Add binstubs to PATH via export PATH=".git/safe/../../bin:$PATH" in ~/.zshenv
> > +mkdir -p .git/safe
>
> Yup, and while I disagree it should be there as well, it's scarier in an
> open source project that might be using tb's dotfiles directly. In a closed
> source project only a few people control it's a more understandable risk to
> take, but in an open source project with more contributors it scares me too
> much.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/thoughtbot/scenic/pull/155/files/9d23c1f64fb4d03f2c7f45ab328d4ecef3e1df2d#r65708461>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAJSWNNkCpv5MXI18aZxXg9ZNn7C8SOEks5qIC8fgaJpZM4ItiJ->
> .
>